### PR TITLE
Reorganise profile settings

### PR DESCRIPTION
**Pull request by SirSaltyy**

### CODE REVIEW

**Comment by SirSaltyy** [comment]
Something went weird when I tried to merge main into my branch... and now it thinks I've updated every single file that got changed in main in the two weeks since I first created my branch lol... and the conflicts that I resolved locally didn't seem to update

File: web/components/buttons/edit-profile.tsx [hunk 1]
```diff
@@ -0,0 +1,208 @@
+import { PrivateUser, User } from 'common/user'
```
Comment (jahooma) [inline web/components/buttons/edit-profile.tsx:1]: This file should be moved under the `profile` directory

File: web/pages/sitemap.tsx [hunk 2]
```diff
@@ -105,7 +105,10 @@ export default function AboutPage() {
               title="⚙️ Notification & email settings"
               href="/notifications?tab=settings"
             />
-            <LabCard title="✏️ Edit profile" href="/profile" />
+            <LabCard
```
Comment (jahooma) [inline web/pages/sitemap.tsx:108]: Only show this card if the user is logged in. I.e. `{user && <LabCard ...}`

**Review by jahooma — APPROVED** [review]
Code looks good! I'm trusting that UI updates also look good.

Seems like an improvement in UX to have the edit tab in the settings popup.

**Pull request merged**